versioned_docs/version-8.2/components/modeler/web-modeler/play-your-process.md

```diff
@@ -61,10 +61,10 @@ Incidents are raised as they would be in Zeebe. Here are some common incidents a
 
 ### Limitations and availability
 
-Play uses the most recent alpha version of Zeebe available. Any BPMN elements not available in the most recent alpha release will not be available in Play.
-In addition, **inbound Connectors** are not supported.
+Play uses the most recent alpha version of Zeebe available. Any BPMN elements unavailable in the most recent alpha release will not be available in Play.
+**Inbound Connectors** do not connect to external systems and must be completed manually. **Start events with forms** will be completed without the form being shown.
```

Comment:
This would be good set aside as an :::admonition like

::: note
**Inbound Connectors** do not connect to external systems and must be completed manually. **Start events with forms** will be completed without the form being shown.
:::

Reply:
Coule be useful to link to the docs on available inbound Connectors and start events/forms too